science/ldas-tools-al/Portfile

```diff
@@ -34,14 +35,17 @@ configure.args PYTHON=false \
                --without-dot \
                --disable-latex
 
-#------------------------------------------------------------------------
+if {${configure.cxx} eq "macports-clang-3.7" } {
+    configure.cxx_stdlib "libc++"
+}
 
 if {${configure.cxx_stdlib} eq "libstdc++" } {
     configure.args-append --disable-cxx11
 }
 
-# requires clang from Xcode5 or higher to build
+# requires clang from Xcode 5.0 or higher to build
 compiler.blacklist-append {clang < 500.2.75} llvm-gcc-4.2 gcc-4.2
+compiler.blacklist-append macports-clang-3.6  macports-clang-3.5 macports-clang-3.4
```

Comment:
I don't understand why these clang compilers would be blacklisted. In particular `macports-clang-3.4` works just fine on 10.7 and you could in fact get `macports-clang-3.3` with the above list.

I'm not sure, but is `macports-clang-3.7` actually on the fallback list?